src/test/Makefile

```diff
@@ -1,12 +1,47 @@
-TARGETSS := all verify clean cleanall cleanlift recompile json cleanjson cleangts gts
+TARGETSS := all verify repro-stash repro-check md5sum-check md5sum-update clean cleanall cleanlift recompile json cleanjson cleangts gts
 
-SUBTARGETS = $(wildcard correct/*/ incorrect/*/)
-.PHONY : $(TARGETSS) $(SUBTARGETS) correct incorrect
+# subdirectories of src/test. to be entered into by this makefile.
+DIRS := correct incorrect extraspec_correct extraspec_incorrect indirect_calls memory_regions procedure_summaries dsa irreducible_loops
```

Comment:
dsa and irreducible_loops are not currently set up to be lifted with the same scripts as everything else

Reply:
i rearranged their directory structure so they worked. if this is not appropriate, i can revert those changes.

Reply:
I don't think it's broken those tests as much as just created a bunch of extra files that are not used by those test suites. I can't say whether those new files would pass or not but most would be redundant.

Reply:
I think the paths do need updating, because the .adt/etc files have moved into new compiler-specific subdirectories. I'm also not sure which compiler was used to produce the previous files.

Reply:
Yes, but you haven't removed the old files so that hasn't actually broken anything.

I can safely say that the DSA tests will not pass if you just switch out the input files though.

I don't know which compiler was used to produce the previous files either, those aren't my tests, but most of the compiler option-specific versions for the DSA tests and the irreducible loops tests would be redundant.

Reply:
Oops, you're right, I forgot to delete the old files. I'll undo those changes for now so they can be given more attention.